File: Awesome.AI.Source/Awesome.AI/Common/MyStats.cs
using System.Text.Json;

namespace Awesome.AI.Common
{
    //public class Stat
    //{
    //    public string _name { get; set; }
    //    public double _index { get; set; }
    //    public int hits {  get; set; }       
    //}

    public class MyStats
    {
        public MyStats()
        {
            hits = new Dictionary<int, int>();
            units = new Dictionary<int, int>();
        }

        public Dictionary<int, int> hits { get; set; }
        public Dictionary<int, int> units { get; set; }

        public string HitsJson()
        {
            string json = JsonSerializer.Serialize(hits);

            return json;
        }

        public string UnitsJson()
        {
            string json = JsonSerializer.Serialize(units);

            return json;
        }
    }

    public class MyMeters
    {
        public int units_added {  get; set; }
        public int units_removed { get; set; }
    }
}

File: Awesome.AI.Python/Common/MyStats.py
from dataclasses import dataclass
import json


@dataclass
class MyMeters:
    units_added: int = 0
    units_removed: int = 0


class MyStats:
    def __init__(self):
        self.hits = {}
        self.units = {}
    def HitsJson(self): return json.dumps(self.hits, separators=(",", ":"))
    def UnitsJson(self): return json.dumps(self.units, separators=(",", ":"))
